docs/book/src/clusterctl/provider-contract.md

```diff
@@ -339,6 +339,8 @@ providers.
 | CAPOSC        | cluster.x-k8s.io/provider=infrastructure-outscale     |
 | CAPK0S        | cluster.x-k8s.io/provider=infrastructure-k0smotron    |
 | CAIPAMIC      | cluster.x-k8s.io/provider=ipam-in-cluster             |
+| CAIPAMX       | cluster.x-k8s.io/provider=ipam-nutanix                |
+| CAREN         | cluster.x-k8s.io/provider=runtime-extensions-nutanix  |
```

Comment:
I'm surprised in one that it uses X for the provider identifier and the other uses N, should these not be consistent?

Reply:
I'm from the UK where carex is a major brand of soap 🤣 but you're right it would make sense to rename it to be consistent. I'll get on to that when I get a chance.